python/dgl/data/gnn_benckmark.py

```diff
@@ -42,17 +43,17 @@ def process(self):
         self._print_info()
 
     def has_cache(self):
-        graph_path = os.path.join(self.save_path, 'dgl_graph.bin')
+        graph_path = os.path.join(self.save_path, 'dgl_graph_fix.bin')
         if os.path.exists(graph_path):
             return True
         return False
 
     def save(self):
-        graph_path = os.path.join(self.save_path, 'dgl_graph.bin')
+        graph_path = os.path.join(self.save_path, 'dgl_graph_fix.bin')
         save_graphs(graph_path, self._graph)
 
     def load(self):
-        graph_path = os.path.join(self.save_path, 'dgl_graph.bin')
+        graph_path = os.path.join(self.save_path, 'dgl_graph_fix.bin')
```

Comment:
Discussed with @hetong007 , to better track the change. Let's use version number in the cache name. For example, here you should use `'dgl_graph_v1.bin'`. The implicit rule is if there's any code changes to the preprocess logic which will invalidate user's cached data, use a new cache name by increasing the version number of one. cc @sneakerkg @HuXiangkun @classicsong for visibility

Reply:
Addressed